cluster/config-defaults.yaml

```diff
@@ -220,12 +220,15 @@ teapot_admission_controller_daemonset_reserved_memory: "64Gi"
 {{if eq .Environment "production"}}
 teapot_admission_controller_validate_application_label: "true"
 teapot_admission_controller_validate_pod_template_resources: "true"
+teapot_admission_controller_preemption_enabled: "true"
 {{else if eq .Environment "e2e"}}
 teapot_admission_controller_validate_application_label: "false"
 teapot_admission_controller_validate_pod_template_resources: "false"
+teapot_admission_controller_preemption_enabled: "true"
 {{else}}
 teapot_admission_controller_validate_application_label: "false"
 teapot_admission_controller_validate_pod_template_resources: "true"
+teapot_admission_controller_preemption_enabled: "true"
```

Comment:
Should this also be enabled in test? Considering we are switching to `cluster-critical-nonpreempting` this could also be `false` or?

Reply:
Oops. Will fix!